tests/auto/runtests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+BASEDIR=$(dirname $0)
+PWD=`(cd $BASEDIR && pwd)`
+
+export LD_LIBRARY_PATH=$PWD/../../../qtmozembed/objdir-mer-dbg/src
```

Comment:
This script is supposed to run in a development environment where we always have a qtmozembed build and tst_* apps are not installed to their respective targets. Its purpose is to produce code coverage statistics in build directories.